### ISSUE TITLE
Another dead jenkins saltstack link in tutorial writing tests page

### ISSUE THREAD

**Issue by alan-cugler** [saltstack/salt#56544]
Relevant salt docs pages:
- https://docs.saltstack.com/en/master/topics/tutorials/writing_tests.html

Relevant salt docs file and error:
```
writing output... [ 99%] topics/tutorials/writing_tests

(line  484) broken    https://jenkins.saltstack.com - HTTPSConnectionPool(host='jenkins.saltstack.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f9ae7827940>: Failed to establish a new connection: [Errno -2] Name or service not known',))
```
Comments
- manually verified

**Comment by ggiesen** [comment]
I believe this is resolved by #56210, and this is issue is basically a dupe of #56209

**Comment by barbaricyawps** [comment]
In light of @ggiesen 's comment, I am closing this issue.
